apps/complaints/templates/complaints_all.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html'%}
+{% load static %}
+
+{% block title %}Reclamaciones{% endblock %}
+
+{% block extrahead %}
+<link rel="stylesheet" href="{% static "base.css" %}">
+<link rel="stylesheet" href="{% static "complain_list.css" %}">
+{% endblock %}
+
+{% block content %}
+    {% if user.is_superuser %}
+            {% if complaints %}
+                <div class="hero-add">
+                    <h1>Reclamaciones</h1>
+                </div>
+                {% for complaint in complaints %}
+                    <div class="card">
+                        <div class="title-date">
+                            <h3 class="title">{{ complaint.title }}</h3>
+                            <div class="status-date">
+                                {% if complaint.status == "Abierto" %}
+                                    <h3 class="available-text">Abierto</h3>
+                                {% else %}
+                                    <h3 class="not-available-text">Solucionado</h3>
+                                {% endif %}
+                                <h6>{{ complaint.date|date:"d/m/Y H:i" }}</h6>
+                            </div>
+                        </div>
```

Comment:
It would be nice if you could add the user that is making the complaint and his/her email